WeScan/Scan/ScannerViewController.swift

```diff
@@ -115,6 +115,8 @@ final class ScannerViewController: UIViewController {
     // MARK: - Actions
     
     @objc private func captureImage(_ sender: UIButton) {
+        (navigationController as? ImageScannerController)?.flashToBlack()
+        shutterButton.isUserInteractionEnabled = false
```

Comment:
It might be good to also re-enable user interaction after the flash, even if the view is inaccessible after capturing an image. If this is changed later it could lead to issues.

Reply:
We reenable the user interaction later when we get back the delegate funcrtions `captureSessionManager(_ captureSessionManager: CaptureSessionManager, didFailWithError error: Error)` and `captureSessionManager(_ captureSessionManager: CaptureSessionManager, didCapturePicture picture: UIImage, withQuad quad: Quadrilateral?)`. That should be enough, right?
We don't really get feedback when the flash is done.

Reply:
Ah ok, I did not see that